test/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/request/UpdateVatSubscriptionSpec.scala

```diff
@@ -17,25 +17,44 @@
 package uk.gov.hmrc.vatsubscription.models.updateVatSubscription.request
 
 import uk.gov.hmrc.vatsubscription.assets.TestUtil
-import play.api.libs.json.Json
+import play.api.libs.json.{JsValue, Json}
+import uk.gov.hmrc.vatsubscription.helpers.{DeclarationTestConstants, DeregistrationInfoTestConstants}
 import uk.gov.hmrc.vatsubscription.helpers.UpdateVatSubscriptionTestConstants._
 import uk.gov.hmrc.vatsubscription.models.{MTDfBExempt, MTDfBMandated}
 
 class UpdateVatSubscriptionSpec extends TestUtil {
 
+  val updateVatSubscriptionLatestDESApi1365JsonMax: JsValue = Json.obj(
```

Comment:
As mentioned in an earlier comment, the writes in `UpdateVatSubscription` is not correct for the `commsPreference` field. If after fixing the writes you update this JSON here ("JsonMax" here is referring to every possible field being populated) to include the field `commsPreference`, then the test below will then be properly checking that the field is being written correctly.